launch/tier4_perception_launch/launch/object_recognition/detection/radar_based_detection.launch.xml

```diff
@@ -20,10 +19,17 @@
     <arg name="velocity_threshold" value="$(var filter/velocity_threshold)"/>
   </include>
 
-  <include file="$(find-pkg-share detected_object_validation)/launch/object_lanelet_filter.launch.xml">
-    <arg name="input/object" value="noise_filtered_objects"/>
-    <arg name="output/object" value="lanelet_filtered_objects"/>
-    <arg name="filtering_range_param" value="$(var radar_lanelet_filtering_range_param)"/>
+  <!-- Filter -->
+  <include if="$(eval &quot;'$(var objects_filter_method)'=='position_filter'&quot;)" file="$(find-pkg-share detected_object_validation)/launch/object_lanelet_filter.launch.xml">
+    <arg name="input/object" value="objects_before_filter"/>
+    <arg name="output/object" value="$(var output/objects)"/>
+    <arg name="filtering_range_param" value="$(var object_recognition_detection_object_lanelet_filter_param_path)"/>
+  </include>
+
+  <include if="$(eval &quot;'$(var objects_filter_method)'=='position_filter'&quot;)" file="$(find-pkg-share detected_object_validation)/launch/object_position_filter.launch.xml">
```

Comment:
[Q] There is another filter prepared that filters radar objects based on the distance.
What is the reason for adding this switching option of radar object filtering by position?
Besides, the input/output topic name is not matched to output of above/input of below nodes.

Reply:
@miursh I want to control object filter mode (lanelet or position) as well over autoware_launch. As far as I know, are the input and output topic names of the lanelet_filter and position_filter is the same? By default, lanelet_filter was used in radar based detection.

Reply:
@miursh Thank you for your reply, I get it what you mean on radar side you are right :+1:. So, I removed option on radar side (There is only lanelet filter as in current version of autoware).

Reply:
@miursh friendly ping